TombEngine/Game/camera.cpp

```diff
@@ -1406,6 +1530,11 @@ void CalculateCamera()
 		return;
 	}
 
+	if (ItemCameraOn == true)
```

Comment:
`if (ItemCameraOn)` is enough for this statement, you don't have to explicitly say `== true` here.

Reply:
fixed